sql_generators/glean_usage/templates/baseline_clients_daily_v1.query.sql

```diff
@@ -33,6 +33,7 @@ WITH base AS (
     COALESCE(client_info.locale, metrics.string.glean_baseline_locale) AS locale,
     metadata.geo.city,
     metadata.geo.country,
+    metadata.isp.`name` AS isp,
```

Comment:
Fixed this path.

Reply:
`name` was being highlighted in my editor as a keyword, so it's probably a keyword for some SQL dialect. But it's not in [BigQuery's list of reserved keywords](https://cloud.google.com/bigquery/docs/reference/standard-sql/lexical#reserved_keywords) so dropping the backticks should be fine.